Automaton/ScrapNewsfromIndiaToday.ipynb

```diff
@@ -173,98 +105,98 @@
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>0</th>\n",
-       "      <td>Covid-infected cop falls to death, third suspi...</td>\n",
-       "      <td>https://www.indiatoday.in/india/story/covid-in...</td>\n",
-       "      <td>Prior to head constable Diwakar Sharma, two Co...</td>\n",
-       "      <td>[A police constable who had tested positive fo...</td>\n",
+       "      <td>Chirag Paswan a villain in NDA? BJP misses sen...</td>\n",
+       "      <td>https://www.indiatoday.in/elections/story/chir...</td>\n",
+       "      <td>Lok Janshakti Party (LJP), led by Chirag Paswa...</td>\n",
+       "      <td>[Lok Janshakti Party (LJP), led by Chirag Pasw...</td>\n",
```

Comment:
Please clear the output fields in your notebook before we merge this PR.  It makes this repository very large if we have a lot of outdated static text that your program can easily recreate.